spec/forms/controlled_work_document_selection_spec.rb

```diff
@@ -65,4 +65,14 @@
     click_on "Start another eligibility check"
     expect(page).to have_content "What level of help does your client need?"
   end
+
+  context "when MTR phase 1 is in effect" do
+    before { travel_to Date.new(2023, 8, 4) }
+
+    it "downloads an updated PDF" do
+      choose "CW1&2 - mental health"
+      click_on "Download the pre-populated form"
+      expect(page.response_headers["Content-Type"]).to eq("application/pdf")
```

Comment:
i might be missing something here, how do we know this is the new file?

Reply:
Good point. Mostly this spec was testing "when we load an MTR phase 1 PDF, nothing explodes" rather than "when we're after 3 August, we load an MTR phase 1 PDF". I've added a new expectation to specify that the correct mapping file gets retrieved.